crates/networking/src/networking.rs

```diff
@@ -506,25 +544,49 @@ impl Network for SubNetwork {
     }
 }
 
+/// Serializes an object to a JSON byte vector.
+///
+/// # Arguments
+/// * `object` - The object to serialize that implements the `Serialize` trait
+///
+/// # Returns
+/// * `Result<Vec<u8>, serde_json::Error>` - The serialized bytes on success
+///
+/// # Errors
+/// * Returns a `serde_json::Error` if serialization fails due to invalid data structures
+///     or other JSON serialization errors
+pub fn serialize(object: &impl Serialize) -> Result<Vec<u8>, serde_json::Error> {
+    serde_json::to_vec(object)
```

Comment:
Using bincode and serde json, serialize fn could have an explicit name on the dpeendency?